app/AcquireIntersonArrayRF.cxx

```diff
@@ -119,20 +119,20 @@ int main( int argc, char * argv[] )
   hwControls.DisableHardButton();
 
   ContainerType container;
+  container.SetHWControls(&hwControls);
 
   const int height = hwControls.GetLinesPerArray();
```

Comment:
Hard call - the drivers and documentation refer to height and width.   I hesitate to deviate too far from the C# because it could create communication/documentation/support issues.